decentraland/unity-renderer
# Fix: ff changes and bugfixes for notifications and channels

PR to fix different behavior in the notifications panel and channels integration.

To test:
* Verify that the notification panel appears without using the ENABLE_NOTIFICATION_CHAT flag in the URL
_______
* Verify that scrolling in a chat panel the old messages are loaded correctly without resetting the scroll to the bottom
_______
* Open different panels 
* Verify that the navigation between panel is correct (every chat should close on the X icon, and should go back to the conversation list on the back button, they should all close when locking the cursor in game)
* Verify that the notifications panel appears when closing any chat panel and that the background hides when not mouse hovering
